draft-ietf-spice-sd-cwt.md

```diff
@@ -196,122 +279,78 @@ sd-protected = {
-  bstr ;.size 16,     ; 128-bit salt
-  any                ; claim value
-]
-~~~
+# SD-CWT Presentation
 
-## Creating a Key Binding Token {#kbt}
+When a Holder presents an SD-CWT to a Verifier, it can disclose
+none, some, or all of its blinded claims. If the Holder wishes to disclose any claims, it includes those Salted Disclosed Claims in the `sd_claims` claim of the unprotected header.
 
 ~~~ cddl
+sd-cwt-presentation = #6.18([
+   protected: bstr .cbor sd-protected,
+   unprotected-presentation,
+   payload: bstr .cbor sd-payload,
+   signature: bstr
+])
 
-digested-salted-disclosed-claim = bstr;
-salted-disclosed-claim = salted-claim / salted-element
-salted-claim = [
-  bstr .size 16,  ; 128-bit salt
-  (int / text),   ; claim name
-  any             ; claim value
-]
-salted-element = [
-  bstr .size 16, ; 128-bit salt
-  any            ; claim value
-]
-sd-cwt-cnf = COSE_Key / Encrypted_COSE_Key / kid
-sd-cwt = [
-  protected,
-  unprotected: {
-    ?(sd_claims: TBD1): bstr .cbor [ + salted-disclosed-claim ],
-    ?(sd_kbt: TBD2): bstr .cbor sd-cwt-kbt,
-  },
-  payload :  bstr .cbor {
-    ?(iss: 1) => tstr, ; "https://issuer.example"
-    ?(sub: 2) => tstr, ; "https://device.example"
-    ?(aud: 3) => tstr, ; "https://verifier.example"
-    ?(exp: 4) => int,  ; 1883000000
-    ?(nbf: 5) => int,  ; 1683000000
-    ?(iat: 6) => int,  ; 1683000000
-    &(cnf: 8) => sd-cwt-cnf,  ; 1683000000
-
-    ?(sd_alg: TBD4) => int,             ; -16 for sha-256
-    ?(redacted_keys: TBD5) => [         ; redacted map keys
-      digested-salted-disclosed-claim
-    ],
-
-    ; redaction in an example map value that is an array
-    &(example-array-key: -65537) => [
-      123,
-      { ; redacted array element
-        &(redacted_element: TBD6) =>
-        digested-salted-disclosed-claim
-      },
-      789,
-      { ; redacted array element
-        &(redacted_element: TBD6) =>
-        digested-salted-disclosed-claim
-      },
-    ]
-  }
-  signature : bstr,
-]
+unprotected-presentation = {
+   &(sd_kbt: TBD2) ^ => bstr .cbor kbt-cwt,
+   ? &(sd_claims: TBD1) ^ => bstr .cbor salted-array,
+   * key => any
+}
 ~~~
 
-As described above, an SD-CWT is a CWT with claims that require confirmation and support selective disclosure.
-Confirmation mitigates risks associated with bearer token theft.
-Note that new confirmation methods might be registered and used after this document is published.
-Selective disclosure enables data minimization.
-The mechanism through which map keys and array elements are disclosed is different, see SD-CWT Validation for details.
-CWT Claims which are not explictly marked redactable by the Issuer are mandatory to disclose by the Holder.
-A detailed privacy and security analysis of all mandatory and optionally disclosed claims SHOULD be performed prior to issuance.
+As described in the CDDL above, a SD-CWT presentation to a Verifier has the
+same syntax as an SD-CWT issued to a Holder, except for its unprotected header.
```

Comment:
I would suggest saying what about the unprotected header is different.  Help readers understand without having to look elsewhere in the spec.

Reply:
I'll take a stab at this tomorrow.

Reply:
> I'll take a stab at this tomorrow.

How well does the proposed change (starting line 329) address your concern about explaining the unprotected header?